src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -125,7 +125,8 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
         const {
             config: { selectedSegmentItems },
         } = this.props;
-        if (this.chart.getLayer(DEFAULT_LAYER_NAME)) {
+
+        if (selectedSegmentItems !== undefined && this.chart.getLayer(DEFAULT_LAYER_NAME)) {
```

Comment:
this could be shorter 
`selectedSegmentItems && this.chart.getLayer(DEFAULT_LAYER_NAME) `